policies/io/konveyor/forklift/vmware/external_data.rego

```diff
@@ -1,16 +1,21 @@
 package io.konveyor.forklift.vmware
 
-runtime            := opa.runtime()
-provider           := input.provider
-vm_moref           := input.vm_moref
-inventory_hostname := concat(":", [runtime.env["FORKLIFT_INVENTORY_SERVICE_HOST"], runtime.env["FORKLIFT_INVENTORY_SERVICE_PORT"]])
-inventory_url      := concat("/", ["https:/", inventory_hostname, "namespaces", provider.namespace, "providers", "vsphere", provider.name, "workloads", vm_moref])
-vm                 := http.send({"url": inventory_url, "method": "get", "tls_insecure_skip_verify": true}).body
+runtime                         := opa.runtime()
+provider                        := input.provider
+vm_moref                        := input.vm_moref
+forklift_inventory_service_port := runtime.env["FORKLIFT_INVENTORY_SERVICE_PORT"]
+namespace                       := runtime.env["NAMESPACE"]
+ca_tls_cert_file                := runtime.env["CA_TLS_CERTIFICATE"]
+inventory_hostname              := concat(".", ["forklift-inventory", namespace, "svc", "cluster", "local"])
```

Comment:
Any reason for not using a variable? If we change the inventory service name in the operator, we will need to change this file inside the container image, instead of just restart the pod.